lib/preprocessor.js

```diff
@@ -154,6 +190,7 @@ class PreprocessedData {
       // Didn't find default, so infer default from namespace
       const parts = ns.split('.');
       return 'http://standardhealthrecord.org/' + parts.join('/') + '/vs';
+      //TODO: Ask Chris if this should be replaced by config
```

Comment:
Yes, I think this should be replaced to either use the project URL or to use a new config parameter that specifies a VS prefix URI.